cli/src/pcluster/templates/budget_builder.py

```diff
@@ -1,28 +1,39 @@
+import os
+from typing import List
+
 from aws_cdk import aws_budgets as budgets
-from aws_cdk.core import Construct
+from aws_cdk import aws_iam as iam
+from aws_cdk import aws_lambda as lambda_
```

Comment:
Can we rename lambda_ to be awslambda to be aligned with the code in other places?

Reply:
Renamed!